testsuite/integration-arquillian/servers/app-server/tomcat/common/src/main/java/org/keycloak/testsuite/arquillian/tomcat/container/CommonTomcatDeploymentArchiveProcessor.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.keycloak.testsuite.arquillian.tomcat.container;
+
+import org.jboss.arquillian.container.test.spi.client.deployment.ApplicationArchiveProcessor;
+import org.jboss.arquillian.test.spi.TestClass;
+import org.jboss.logging.Logger;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.keycloak.testsuite.adapter.servlet.TomcatConfigApplication;
+import org.keycloak.testsuite.utils.arquillian.DeploymentArchiveProcessorUtils;
+import org.keycloak.testsuite.utils.arquillian.tomcat.TomcatDeploymentArchiveProcessorUtils;
+
+/**
+ *
```

Comment:
Probably it would be useful to either remove the empty javadoc (the same for the rest of files in this PR) or put something meaningful here. But It's a nitpick.

Reply:
There is @author in this comment. Plenty of files have such comment on top of it. I think it is good for others to see who to contact in case of some questions. But I can remove it.

Reply:
Sure, no problem.

As a side note - I always use `git gui blame` or `git annotate` to find the author. Javadocs like that often are a bit misleading (of course in my opinion), since the file changes over the time.

Bu anyway - granted. Please keep it or remove it if you'd like.

Reply:
You are right. Removed. Thanks